dcos/auth.py

```diff
@@ -3,7 +3,6 @@
 import textwrap
 import time
 import webbrowser
-
 import jwt
```

Comment:
The typical convention as of PEP8 is to have three import blocks. The first imports from the standard library (which is everything up to `time` here), the second imports external and established third-party modules (which is `jwt`, but probably also `webbrowser`), and the third imports project-local modules/packages.

Not sure if that convention is adhered to in the CLI code :-)

Reply:
I didn't change this on purpose, reverting.